src/Illuminate/Support/Arr.php

```diff
@@ -439,6 +439,17 @@ public static function pull(&$array, $key, $default = null)
     }
 
     /**
+     * Get a random value from an array.
+     *
+     * @param  array   $array
+     * @return mixed
+     */
+    public static function random($array)
+    {
+        return $array[array_rand($array)];
+    }
+
+    /**
```

Comment:
Also, just as a proposal: in addition to the regular "raffle tickets" mode, a "dice roll" mode might be added, allowing items to be picked several times. (see https://github.com/fzaninotto/Faker/pull/1060 to better understand what I mean)